.github/workflows/rust-av.yml

```diff
@@ -108,3 +119,30 @@ jobs:
     - name: Run doc
       run: |
         cargo doc --workspace --all-features --verbose
+
+
+  wasm-test:
+    # runtime is normally 2 minutes
+    timeout-minutes: 10
```

Comment:
Is there a specific reason that the wasm-test job could hit this timeout?

Reply:
Oops, leftover. Removed!